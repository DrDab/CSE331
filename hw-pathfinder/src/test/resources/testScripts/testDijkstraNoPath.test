# Test if graph with no valid path causes correct Dijkstra test output.

# Create graph g
CreateGraph g

# Add city names to graph g
AddNode g LosSantos
AddNode g SanAndreas
AddNode g SanFierro
AddNode g LasVenturas
AddNode g Ludendorff
AddNode g Vice
AddNode g Liberty
AddNode g Alderney

# Add edges to graph g
AddEdge g LosSantos SanAndreas 50.0

# Test pathfinding
FindPath g LosSantos Liberty
