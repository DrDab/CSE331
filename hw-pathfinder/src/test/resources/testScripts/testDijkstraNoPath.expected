# Test if graph with no valid path causes correct Dijkstra test output.

# Create graph g
created graph g

# Add city names to graph g
added node LosSantos to g
added node SanAndreas to g
added node SanFierro to g
added node LasVenturas to g
added node Ludendorff to g
added node Vice to g
added node Liberty to g
added node Alderney to g

# Add edges to graph g
added edge 50.000 from LosSantos to SanAndreas in g

# Test pathfinding
path from LosSantos to Liberty:
no path found
